openedx/edx-cookiecutters
# feat: Switch from edx-sphinx-theme to sphinx-book-theme

The edx-sphinx theme is being deprecated, and replaced with sphinx-book-theme. This removes references to the deprecated theme and replaces them with the new standard theme for the platform.

It also removes some unnecessary overrides from the subtemplates.

*Testing Steps:*

- Follow the project's instructions to create the different types of projects. 
- Ensure that the projects created use the new Sphinx theme, and don't contain other differences that could break things. 

Here is one rough process:

- Checkout the master branch. 
- Run a command like `make cookiecutter-django-app`
- Keep the default name or provide a name for the app (for eg. `app`)
- The project files will be rendered in the var directory (for eg. var/app)
- Rename that folder to something like `app.old`
- Checkout the branch from this PR and repeat the above steps and use the same app name. 
- Using the same app name means that it should create the same structure, and the only differences should be from the change in templates in this PR. 
- Look at the diff and make sure it aligns with the goals of this PR. 
- Repeat for all templates (cookiecutter-django-ida, cookiecutter-python-library, cookiecutter-xblock).

ref: https://github.com/openedx/edx-sphinx-theme/issues/193

**Merge checklist:**
Check off if complete *or* not applicable:
- [x] Noted any: Concerns, dependencies, deadlines, tickets
